app/views/manifestation/_browse_list.html.haml

```diff
@@ -31,18 +31,28 @@
         #custom-select{title: t(:worklist_sort_tt)}
           %select#sort_by_dd{name: 'sort_by'}
             %optgroup{label: t(:alefbet)}
-              %option{value: 'alphabetical_asc', selected: @sort == 'alphabetical_asc' ? 'true': nil}= t(:alefbet_asc)
-              %option{value: 'alphabetical_desc', selected: @sort == 'alphabetical_desc' ? 'true': nil}= t(:alefbet_desc)
+              %option{value: 'alphabetical_asc', selected: @sort == 'alphabetical_asc' ? 'true': nil}
+                = t(:alefbet_asc)
+              %option{value: 'alphabetical_desc', selected: @sort == 'alphabetical_desc' ? 'true': nil}
+                = t(:alefbet_desc)
             %optgroup{label: t(:popularity_group)}
-              %option{value: 'popularity_desc', selected: @sort == 'popularity_desc' ? 'true': nil}= t(:popularity_desc)
-              %option{value: 'popularity_asc', selected: @sort == 'popularity_asc' ? 'true': nil}= t(:popularity_asc)
-            %optgroup{label: t(:dates)}
-              %option{value: 'publication_date_asc', selected: @sort == 'publication_date_asc' ? 'true': nil}= t(:pubdate_asc)
-              %option{value: 'publication_date_desc', selected: @sort == 'publication_date_desc' ? 'true': nil}= t(:pubdate_desc)
-              %option{value: 'creation_date_asc', selected: @sort == 'creation_date_asc' ? 'true': nil}= t(:workdate_asc)
-              %option{value: 'creation_date_desc', selected: @sort == 'creation_date_desc' ? 'true': nil}= t(:workdate_desc)
-              %option{value: 'upload_date_asc', selected: @sort == 'upload_date_asc' ? 'true': nil}= t(:uploaddate_asc)
-              %option{value: 'upload_date_desc', selected: @sort == 'upload_date_desc' ? 'true': nil}= t(:uploaddate_desc)
+              %option{value: 'popularity_desc', selected: @sort == 'popularity_desc' ? 'true': nil}
+                = t(:popularity_desc)
+              %option{value: 'popularity_asc', selected: @sort == 'popularity_asc' ? 'true': nil}
+                = t(:popularity_asc)
+            %optgroup{ label: t(:dates) }
+              %option{value: 'publication_date_asc', selected: @sort == 'publication_date_asc' ? 'true': nil}
+                = t(:pubdate_asc)
+              %option{value: 'publication_date_desc', selected: @sort == 'publication_date_desc' ? 'true': nil}
+                = t(:pubdate_desc)
+              %option{value: 'creation_date_asc', selected: @sort == 'creation_date_asc' ? 'true': nil}
+                = t(:creation_date_asc)
```

Comment:
I've also renamed i18n string from workdate_desc to creation_date_desc (and same for _asc), to match naming in ruby code.